cpp/src/parquet/encoding.cc

```diff
@@ -43,6 +44,11 @@ using arrow::internal::checked_cast;
 
 namespace parquet {
 
+template <typename StatusReturnBlock>
+void ThrowNotOk(StatusReturnBlock&& b) {
+  PARQUET_THROW_NOT_OK(b());
+}
```

Comment:
Might be worth moving this to some reusable place at some point

Reply:
will do